packages/dev/serializers/test/integration/glTFSerializer.test.ts

```diff
@@ -555,5 +554,92 @@ describe("Babylon glTF Serializer", () => {
                 expect(node.mesh).toEqual(0);
             }
         });
+        it("should not export a root conversion node", async () => {
+            const assertionData = await page.evaluate(() => {
+                return BABYLON.SceneLoader.AppendAsync("https://models.babylonjs.com/Tests/TwoQuads/", "TwoQuads.gltf", window.scene).then((scene) => {

```

Comment:
Is it `models.babylonjs.com` or `assets.babylonjs.com`? There is both in this file. Which is the right one?

Reply:
Everything in `models.babylonjs.com` is available in `assets.babylonjs.com/meshes/`, and `assets` is preferred, so I've updated this.